src/Employer/Employer.Web/Controllers/Part2/VacancyDescriptionController.cs

```diff
@@ -44,6 +45,8 @@ public async Task<IActionResult> VacancyDescription(VacancyDescriptionEditModel
                 return View(vm);
             }
             
+            //if isFaaV2ImprovementsEnabled - need to redirect to new page.
```

Comment:
Maybe this comment can come out? I am not sure if it's referring to functionality that isn't there, or referring to a future piece of work, or what.